internal/model/microservice.go

```diff
@@ -3,3 +3,8 @@ package model
 type MicroserviceResponse struct {
 	Content string `json:"text_description"`
 }
+
+type MicroservicePromptResponse struct {
+	CheckResult bool   `json:"check_result"`
+	Content     string `json:"text_description"`
```

Comment:
the microservice is supposed to return a "check_result" only, returning the content is another costly operation that is not really required